app/views/applications/show.html.erb

```diff
@@ -9,3 +9,16 @@
   <p>Status: <%= @application.status %></p>
 
   <p>Applicant: <%= link_to @application.developer.first_name + ' ' + @application.developer.last_name, developer_path(@application.developer) %></p>
```

Comment:
I'm also going to make this a string interpolation, but I'll do this myself on the pundit branch if that's okay